src/panel.html

```diff
@@ -1,15 +1,264 @@
-<html>
+<html class="-theme-with-dark-background">
   <head>
     <style>
+      body {
```

Comment:
Just curious - how is this constructed? The style seems very sophisticated.

Reply:
These are excerpts from the following:

- https://github.com/ChromeDevTools/devtools-frontend/blob/main/front_end/ui/legacy/inspectorCommon.css#L81-L150
- https://github.com/ChromeDevTools/devtools-frontend/blob/main/front_end/ui/legacy/themeColors.css#L13-L336

I've been thinking about how to manage these. (Probably namely comments referring to those sources, but also to maybe put it in their own files and either `<link>` them or inject them in a build step.) I also want to pair them down more. I removed some colors already, and want to remove most of the rest. I think pair down the fonts or remove them altogether and use `sans-serif` alone.

Reply:
Since I've merged this change I made an issue to help track this https://github.com/GoogleChrome/audion/issues/126.